fix: add outlined for username login input and fix eye icon padding

ykhedher opened the pull request:
![Screen Shot 2022-01-09 at 12 11 31 PM](https://user-images.githubusercontent.com/47088512/148679769-4adfe881-bf76-425e-8c0f-a96e1cc291d6.png)

FredrikOseberg commented:
> ![Screen Shot 2022-01-09 at 12 11 31 PM](https://user-images.githubusercontent.com/47088512/148679769-4adfe881-bf76-425e-8c0f-a96e1cc291d6.png)

Why is the username / email input now a single line input field?

ykhedher commented:
> > ![Screen Shot 2022-01-09 at 12 11 31 PM](https://user-images.githubusercontent.com/47088512/148679769-4adfe881-bf76-425e-8c0f-a96e1cc291d6.png)
> 
> Why is the username / email input now a single line input field?

@FredrikOseberg This is actually the error I fixed, I think I have messed it it up when I added the password field. This pr fix it ( make it outlined), sorry for the confusion.

FredrikOseberg reviewed (approved):
LGTM
